sdk/formrecognizer/Azure.AI.FormRecognizer/src/AddressValue.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis
```

Comment:
Why create this class at all? The generated model should suffice no?

Reply:
CodeGen generates `public` models, but we added a transform to default everything to `internal` ([autorest.md file](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/src/autorest.md#make-generated-models-internal-by-default)):
```
directive:
  from: https://github.com/Azure/azure-rest-api-specs/blob/7043b48f4be1fdd40757b9ef372b65f054daf48f/specification/cognitiveservices/data-plane/FormRecognizer/stable/v2.1/FormRecognizer.json
  where: $.definitions.*
  transform: >
    $["x-accessibility"] = "internal"
```

We started doing this because we had way more generated models we wanted to keep hidden than models we wanted to expose. I can't tell for sure if that's still true today (would need to double check).

Reply:
We are adding this file to tell CodeGen this model should be made `public` instead of defaulting to `internal`.

Reply:
I recently learned that something other libraries have done for classes that are just exposed but don't have any logic to it, is to aggregate them in one file.
We did this in TA https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/GeneratorStubs/InternalTypes.cs (although it was to hide things, but same concept) and I think KV has it too? or Search?.

Something to consider for future reference :)